src/app/project/project.component.spec.ts

```diff
@@ -86,10 +99,13 @@ describe('ProjectComponent', () => {
       expect(component).toBeTruthy();
     });
 
-    it('should create when no project data is available', () => {
-      expect(component.project).toBeUndefined();
-      expect(component.builds).toBeUndefined();
-    });
-  });
+    // TODO - Radu M
```

Comment:
Shall we create an issue around this TODO for a follow-up (if still applicable)?  Or shall we remove?

Reply:
Yeah, I'd like to tackle this in a follow-up PR.